projects/chaos/src/sequencer_fsm.c

```diff
@@ -15,121 +16,129 @@
 // Statics
 static FSM s_sequencer_fsm;
 static SequencerStorage s_storage;
+static bool s_pending;
+static ChaosEventSequence s_current_sequence;
+static ChaosEventSequence s_next_sequence;
 
 // Event order declarations
 
 // Emergency State Order:
 // - Turn off the charger (if active)
-// - Disable DCDC Monitoring
 // - Open main power relay
 // - Open solar master rear relay
 // - Open solar master front relay
-// - Open battery relay
 // - Disable all unnecessary GPIO pins
+// - Disable DCDC Monitoring
+// - Open battery relay (DCDC)
 static const SequencerEventPair s_emergency_events[] = {
   { .raise = { .id = CHAOS_EVENT_CHARGER_OPEN, .data = SEQUENCER_EMPTY_DATA },
     .response = SEQUENCER_NO_RESPONSE },
-  { .raise = { .id = CHAOS_EVENT_MONITOR_DISABLE, .data = POWER_PATH_SOURCE_ID_DCDC },
-    .response = SEQUENCER_NO_RESPONSE },
-  { .raise = { .id = CHAOS_EVENT_CLOSE_RELAY, .data = RELAY_ID_MAIN_POWER },
-    .response = { .id = CHAOS_EVENT_RELAY_CLOSED, .data = RELAY_ID_MAIN_POWER } },
-  { .raise = { .id = CHAOS_EVENT_CLOSE_RELAY, .data = RELAY_ID_SOLAR_MASTER_REAR },
-    .response = { .id = CHAOS_EVENT_RELAY_CLOSED, .data = RELAY_ID_SOLAR_MASTER_REAR } },
-  { .raise = { .id = CHAOS_EVENT_CLOSE_RELAY, .data = RELAY_ID_SOLAR_MASTER_FRONT },
-    .response = { .id = CHAOS_EVENT_RELAY_CLOSED, .data = RELAY_ID_SOLAR_MASTER_FRONT } },
-  { .raise = { .id = CHAOS_EVENT_CLOSE_RELAY, .data = RELAY_ID_BATTERY },
-    .response = { .id = CHAOS_EVENT_RELAY_CLOSED, .data = RELAY_ID_BATTERY } },
+  { .raise = { .id = CHAOS_EVENT_OPEN_RELAY, .data = RELAY_ID_MAIN_POWER },
```

Comment:
What does the "main power relay" refer to?

Reply:
Main power refers to motors.

Reply:
Done. It will need to be changed in `codegen-tooling` as well.